sa/_db/migrations/20150904121736_CertificateStatusNotNullManyFields.sql

```diff
@@ -0,0 +1,23 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+ALTER TABLE `certificateStatus` MODIFY `subscriberApproved` tinyint(1) NOT NULL;
+ALTER TABLE `certificateStatus` MODIFY `status` varchar(255) NOT NULL;
+ALTER TABLE `certificateStatus` MODIFY `ocspLastUpdated` datetime NOT NULL;
+ALTER TABLE `certificateStatus` MODIFY `revokedDate` datetime NOT NULL;
```

Comment:
(I would like that change to happen in a follow-up PR, however. Because this stuff is important and there's slightly more schedule risk to swapping to a pointer.)